src/containers/__tests__/FactoryLoader.spec.tsx

```diff
@@ -88,21 +92,25 @@ jest.mock('../../pages/FactoryLoader', () => {
 });
 
 describe('Factory Loader container', () => {
-  const location = 'http://test-location/';
+  const location = 'http://test-location';
   const workspace = createFakeWorkspaceWithRuntime('id-wksp-test');
+  if (workspace.attributes) {
+    workspace.attributes.stackName = 'http://test-location/?policies.create=peruser';
+  }
```

Comment:
How do you know the attribute would be set in a test where you supposed to have it? I think is it's better to do a single-test-specific setup where it's needed, within that particular test.

Reply:
I prepared a dummy store with one default workspace which is used in the one test. Could you explain the benefits of changing it?

Reply:
I have fixed it.